core/src/main/java/org/arquillian/smart/testing/scm/git/GitRunnerProperties.java

```diff
@@ -2,9 +2,9 @@
 
 public class GitRunnerProperties {
 
-    public static final String PREVIOUS_COMMIT = "git.previous.commit";
-    public static final String COMMIT = "git.commit";
-    static final String LAST_COMMITS = "git.last.commits";
+    public static final String PREVIOUS_COMMIT = "git.range.tail";
```

Comment:
Alex you are in holidays... 😂 but you are right, these should be as described in the issue.